src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.ts

```diff
@@ -469,4 +469,11 @@ export class ProgrammingExerciseUpdateComponent implements OnInit {
     validIdeSelection() {
         return this.programmingExercise.allowOnlineEditor || this.programmingExercise.allowOfflineIde;
     }
+
+    isTagName(e: Event): boolean {
```

Comment:
I would prefer another name such as `isEventInsideTextArea`
Also do not use single character variable names such as `e`

Reply:
```suggestion
    isEventInsideTextArea(event: Event): boolean {
```

Reply:
this makes sense. thanks for the suggestion